code/datums/spells/wizard_spells.dm

```diff
@@ -12,10 +12,11 @@
 	proj_icon_state = "magicm"
 	proj_name = "a magic missile"
 	proj_lingering = 1
+	pass_flags = PASSDOOR
 	proj_type = "/obj/effect/proc_holder/spell/inflict_handler/magic_missile"
 
 	proj_lifespan = 20
-	proj_step_delay = 5
+	proj_step_delay = 2
```

Comment:
does this apply to artificers aswell?

Reply:
It shouldn't, they have their own Lesser Magic Missile spell from what I saw.

Reply:
it will effect cult magic missile as the lesser version is a subtype of this. would you change it so it doesnt effect the lesser version please.